vertigo-bot-executor/src/main/java/io/vertigo/chatbot/engine/plugins/bt/command/bot/BotNodeProvider.java

```diff
@@ -261,6 +261,14 @@ public static BTNode switchTopic(final BlackBoard bb, final String topicCode) {
 		};
 	}
 
+	public static BTNode launchJsEvent(final BlackBoard bb, final String eventName) {
+		return sequence(
+				() -> {
+					bb.putString(BBKey.of(BotEngine.BOT_OUT_METADATA_PATH, "/jsEvent"), eventName);
+					return BTStatus.Succeeded;
+				});
+	}
```

Comment:
```suggestion
		return () -> {
			bb.putString(BBKey.of(BotEngine.BOT_OUT_METADATA_PATH, "/jsEvent"), eventName);
			return BTStatus.Succeeded;
		};
	}
```
pas besoin de la séquence comme on a 1 seul noeud dedans